lib/screens/transfer/receive/receive_detail_qr_code/components/receive_paid_success_dialog.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:intl/intl.dart';
+import 'package:seeds/components/custom_dialog.dart';
+import 'package:seeds/components/profile_avatar.dart';
+import 'package:seeds/datasource/local/models/fiat_data_model.dart';
+import 'package:seeds/datasource/remote/model/profile_model.dart';
+import 'package:seeds/datasource/remote/model/transaction_model.dart';
+import 'package:seeds/design/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/utils/double_extension.dart';
+
+class ReceivePaidSuccessArgs {
```

Comment:
what is the benefit of making this data class instead of passing the arguments into the constructor directly ?